### PR TITLE
Adding backtrace damage repair to the squid

### PR DESCRIPTION
**Pull request by julpark-rh**
# Description

adding the suite to the squid

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/squid/cephfs/tier-4_cephfs_recovery.yaml [hunk 1]
```diff
@@ -221,4 +221,9 @@ tests:
       polarion-id: CEPH-83593150
       desc: Validate all extended attributes that are supported in filesystem
       abort-on-fail: false
-
+  - test:
+      abort-on-fail: false
+      desc: "repair_backtrace_by_scrub"
+      module: cephfs_recovery.backtrace_damage_repair_by_scrub.py
+      polarion-id: CEPH-CEPH-83593391
```
Comment (sumabai) [inline suites/squid/cephfs/tier-4_cephfs_recovery.yaml:228]: please correct typo error in polarion-id value

Comment (julpark-rh) [reply suites/squid/cephfs/tier-4_cephfs_recovery.yaml]: fixed

**Review by neha-gangadhar — APPROVED** [review]
lgtm

File: suites/reef/nvmeof/tier-3_nvmeof_librbd_vs_nvme_128k_block_size.yaml [hunk 2]
```diff
@@ -96,7 +95,6 @@ tests:
                     count: 1
                 gw_node: node6          # gateway node
                 initiator_node: node10   # client node
-                install_gw: true
```
Comment (sumabai) [inline suites/reef/nvmeof/tier-3_nvmeof_librbd_vs_nvme_128k_block_size.yaml:99]: We have considered other FG changes into ours and these needs to be skipped from your PR, please do the needful.

**Comment by hkadam134** [comment]
@julpark-rh - you need to rebase your branch and push only your changes, the PR includes other commits from NVMe.
Please fix it.

**Comment by julpark-rh** [comment]
closing this PR to raise new one

**Pull request closed without merging**